sequencer/src/bin/deploy.rs

```diff
@@ -147,17 +150,34 @@ struct Options {
     #[clap(long, env = "ESP_TOKEN_INITIAL_GRANT_RECIPIENT_ADDRESS")]
     initial_token_grant_recipient: Option<Address>,
 
+    /// The blocks per epoch    
+    #[clap(long, env = "ESPRESSO_SEQUENCER_BLOCKS_PER_EPOCH")]
+    blocks_per_epoch: Option<u64>,
+
+    /// The epoch start block
+    #[clap(long, env = "ESPRESSO_SEQUENCER_EPOCH_START_BLOCK")]
+    epoch_start_block: Option<u64>,
+
     #[clap(flatten)]
     logging: logging::Config,
 }
 
+impl Options {
+    fn validate_upgrade_choice(&self) -> anyhow::Result<()> {
```

Comment:
This can be done with clap.

https://github.com/EspressoSystems/espresso-network/blob/3426a6b0c2a7fe4a3bb8775e06e893693f0d5291/sequencer/src/persistence/sql.rs#L188

Reply:
ooop thank you!

https://github.com/EspressoSystems/espresso-network/commit/050be5fbc9a578ff375e8b7e65593dd4a9d546ea